src/Lib/FileStorage.php

```diff
@@ -193,7 +193,7 @@ protected function copyDir(string $from, string $to): bool {
 			if ($file->isDir()) {
 			  // Create dir if it does not exist
 				if (!is_dir($destDir)) {
-					$this->createDir($destDir, $file->getPath());
+					$this::createDir($destDir, $file->getPath());
```

Comment:
Static calls should use ClassName instead of dynamic variable value. Consider using `static::createDir`